redis_go1.7.go

```diff
@@ -0,0 +1,35 @@
+// +build go1.7
```

Comment:
`"go1.7", from Go version 1.7 onward`
https://golang.org/pkg/go/build/

As I understand this build tag covers all version >=1.7